package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "globalforestwatch",
-  "version": "3.0.1",
+  "version": "3.1.0",
```

Comment:
Not sure why this is here?

Reply:
I don't know either 🤷‍♂️

Reply:
1 and 0 swapped places because 1 was tired of being on the window seat?